tabletop/site_sections/tabletop_checkins.py

```diff
@@ -9,7 +9,7 @@ def index(self, session):
             'attendees': _attendees(session)
```

Comment:
As you can see in the title, I renamed ``tabletop.py`` to ``tabletop_checkins.py`` since we now have multiple different tabletop site sections.